### ISSUE TITLE
hypopg is incompatible with PostgreSQL 10

### ISSUE THREAD

**Issue by robstolarz** [dalibo/hypopg#7]
Unfortunately, hypopg is not compatible with the newest version of Postgres. 

RhodiumToad from freenode's #postgresql writes:
```
<RhodiumToad> specifically,  entry->amhasgettuple = OidIsValid(amroutine->amgettuple);   <-- amroutine->amgettuple there is a function pointer, not an oid
<RhodiumToad> so it should be testing it against NULL, not using OidIsValid
<RhodiumToad> the other [issue] is a change in function signature for ProcessUtility
```

Here is a compilation log demonstrating the latter issue: http://ix.io/AEt. Compiled using gcc 7.2.0 with Postgres 10rc1.

**Comment by rjuju** [comment]
Hi,

Indeed I missed that since 65c5fcd353a OidIsValid shouldn't be used anymore, I'll fix it.  However, changes for processUtility should already be handled in HEAD.

Are this logs from tag 1.0.0 or from HEAD?

**Comment by robstolarz** [comment]
It's from tag 1.0.0 as per https://github.com/ankane/dexter#installation, but I believe it still fails on HEAD.

**Comment by rjuju** [comment]
I fixed all the remaining issue I believe (see https://github.com/dalibo/hypopg/commit/191340ebb4a91754c32ad76263cd0146b32d6d17 and https://github.com/dalibo/hypopg/commit/1a65213a9458e1640ff6b9dcc42b95052cfe8dbf).  I didn't create a release yet, so you can use current HEAD. It's not really a problem since this extension won't store anything, so you can freely drop/create the extension without any impact.

Thanks again for the report!

**Comment by rjuju** [comment]
I just released version 1.1.0, so I'm closing this issue.  Feel free to reopen it if you still encounter any problem.
